src/main/java/org/tarantool/jdbc/SQLDatabaseMetadata.java

```diff
@@ -1085,13 +1085,16 @@ public boolean generatedKeyAlwaysReturned() throws SQLException {
     }
 
     @Override
-    public <T> T unwrap(Class<T> iface) throws SQLException {
-        throw new SQLFeatureNotSupportedException();
+    public <T> T unwrap(Class<T> type) throws SQLException {
+        if (isWrapperFor(type)) {
+            return type.cast(this);
+        }
+        throw new SQLNonTransientException("DatebaseMetadata does not wrap " + type.getName());
```

Comment:
Would be more correct to say 'SQLDatebaseMetadata does not wrap...'?

Reply:
I assume the same for other 'does not wrap' messages, but okay, it is minor thing.